pkgs/applications/science/biology/EZminc/default.nix

```
@@ -1,16 +1,16 @@
-{ stdenv, fetchFromGitHub, cmake, libminc, bicpl, itk, fftwFloat, gsl }:
+{ stdenv, fetchFromGitHub, cmake, pkgconfig, libminc, bicpl, itk, fftwFloat, gsl }:
 
 stdenv.mkDerivation rec { pname = "EZminc";
-  name  = "${pname}-2017-08-29";
+  name  = "${pname}-2019-07-25";
```

if it's not an official release, this is the current practice
```suggestion
  name  = "${pname}-unstable-2019-07-25";
```